velodyne_msgs/msg/VelodyneDeskewInfo.msg

```diff
@@ -0,0 +1,4 @@
+# Velodyne Deskew Info.
```

Comment:
This should have a through explanation of what this is, how it should be used, etc.